### ISSUE TITLE
How to repaint when app is running in background tab?

### ISSUE THREAD

**Issue by telent** [omcljs/om#285]
tl;dr can there be a public function to force a redraw instead of waiting for the requestAnimationFrame callback to run?

I'm writing a music player app which uses the `audio` element to play a list of songs.  It listens for the `ended` event to know when to pull the next track off the play queue, but when it's running in a background tab there are often long gaps between tracks and I hypothesize that this is because the om render is not happening in any timely kind of way because (reasonably enough) you've decided that there's usually no point redrawing if the user isn't there to see it.

I hope that my supposition is correct and I've explained what I'm looking for.   If you're _particularly_ bored, 
- the `ended` event handler is here : https://github.com/telent/sledge/blob/master/src-cljs/core.cljs#L197 
- it sends a message on the 'dequeue' channel to this bit: https://github.com/telent/sledge/blob/master/src-cljs/core.cljs#L107 which takes the first track off the queue
- but not until the next render phase does it redisplay the audio player element and point it at the next track:  https://github.com/telent/sledge/blob/master/src-cljs/core.cljs#L204

This is my first ever Om app, so feedback is welcome but be gentle ;-) I already know there's a lot wrong with it ...
